completion_aggregator/core.py

```diff
@@ -23,6 +23,7 @@
 from . import tracking
 from .cachegroup import CacheGroup
 from .models import Aggregator, StaleCompletion
+
```

Comment:
```suggestion
from . import signals
```

If you prefer this style which also avoids circular imports.

Reply:
Thanks I'll try it.